_static/custom.css

```diff
@@ -11,7 +11,7 @@ div.sphinxsidebar .caption-text {
 }
 
```

Comment:
wait, why does it close "that issue"? either create a new one or dont close any, as that one is irrelevant and already closed :D